CompositeUi/src/reducers/Dashboard.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import ActionTypes from './../constants/ActionTypes';
+
+const initialState = {
+  errors: [],
```

Comment:
If we are using a boolean in `DASHBOARD_DATA_FETCH_ERROR` we should be using it here too